client/Cargo.toml

```diff
@@ -9,6 +9,8 @@ tpm-simulator-tests = []
 [dependencies]
 tpm2-rs-base = { workspace = true }
 tpm2-rs-marshalable = { workspace = true }
+tpm2-rs-unionify = { workspace = true }
```

Comment:
Any particular reason the client is adding this? Probably for sizing in tests. In that case please move to dev-dependencies.

Reply:
Moved 👍 🙂